DP1.20008.001/Collection.methodology.txt

```diff
@@ -0,0 +1 @@
+Conductivity of surface water is measured using vented pressure transducers at upstream and downstream sensor sets at wadeable stream sites, on infrastructure near buoys at river sites, and littoral sensor sets at lake sites.  Measurements spatially represent a point and are made at 0.0167 Hz (1 per minute). Data are published as five- and thirty-minute elevation averages.
```

Comment:
Elevation averages? I think the text here might be meant for surface water elevation